APITesters/SwiftAPITester/SwiftAPITester/StoreProductAPI.swift

```diff
@@ -11,6 +11,7 @@ var product: StoreProduct!
 func checkStoreProductAPI() {
     let localizedDescription: String = product.localizedDescription
     let localizedTitle: String = product.localizedTitle
+    let currencyCode: String? = product.currencyCode
```

Comment:
These need to be added in the Obj-C APITester too 😛

Reply:
Ugh! I thought I got them all this time too 🙈